HW1.Rmd

```diff
@@ -81,7 +82,7 @@ model<-lwlywge~college
 output1<- lm(lwlywge~ college, data = pums2)
 summary(output1)
 ```
-The p-value of collegetrue(educated for 16 years) is significant. That is, if one person graduated from college, the log weekly wage is 0.384275 higher than the one graduated from senior high school.
+The p-value of collegetrue(educated for 16 years) is significant. That is, if one person graduated from college, the log weekly wage is 38.4275% higher than the one graduated from senior high school.
```

Comment:
"the log weekly wage is 38.4275% higher than..." is wrong. Should be just 
"the weekly wage is 38.4275% higher than...".